### DISCUSSION TITLE
v0.0.5 - Just a few fixes

### DISCUSSION THREAD

**Discussion by XurxoMF** [XurxoMF/vs-launcher#16] [Announcements & News]
# Download it - [How to download](https://github.com/XurxoMF/vs-launcher/wiki/How-to-install-VS-Launcher)
| OS | Download Link |
| ------------- | --------------- |
| Windows | [vs-launcher-0.0.5-setup.exe](https://github.com/XurxoMF/vs-launcher/releases/download/0.0.5/vs-launcher-0.0.5-setup.exe) |
| Linux | [vs-launcher-0.0.5.AppImage](https://github.com/XurxoMF/vs-launcher/releases/download/0.0.5/vs-launcher-0.0.5.AppImage) |
> If you already have it installed you don't need it, it'll auto update when you open the launcher!

# Description
Yes, that was fast, the last release was only 1 hour ago... not enough testing I suppose.

# Changelog
- **Fixed:** cs localization not working.
- **Fixed:** Lang menu not showing anything if the selected localization is not present.

<hr /><em>This discussion was created from the release <a href='https://github.com/XurxoMF/vs-launcher/releases/tag/0.0.5'>v0.0.5 - Just a few fixes</a>.</em>

**Comment by BradyThe** [comment]
I completely forgot to mention a suggestion: Give top-minimenu-* a link mouse pointer to make it make it (even) more obvious that those are links that can be clicked.

**Reply by XurxoMF** [reply]
It should show it... I changed them from links to onClick so it opens on the browser... that may have happend...

I'll check it out on the next update! Thx for letting me know.

**Reply by BradyThe** [reply]
It doesn't for me on the installed version. I could be misrepresenting the "issue", though - I mean changing the "normal select" pointer to "link select" (windows names).
![image](https://github.com/user-attachments/assets/aa2f156d-9d06-4e97-8a9f-31e82417c784)

**Reply by XurxoMF** [reply]
Nono, I know it doesn't, when I had them as links the pointer was the hand pointing but I had to change them, now they are buttons and it's not showing the hand pointing anymore xD

Btw, I'll ask other translators too but, How do you prefer I notify you of new translation keys? Through a Discord server with a role ping, a post here on GitHub like the 0.0.4 new keys...(i deleted it bc it's already released)?

**Reply by BradyThe** [reply]
> Btw, I'll ask other translators too but, How do you prefer I notify you of new translation keys? Through a Discord server with a role ping, a post here on GitHub like the 0.0.4 new keys...(i deleted it bc it's already released)?

I personally don't use Discord too much.
A Github ping I will eventually see. I would start checking Github more often, if people would start pinging on here.
I like checking the ModDB fairly regularly for new or updated mods, so including a remark like "Added: New translation strings" would currently get my attention probably quickest.

**Pinging on Discord/Github still makes most sense. Ideally you want files not lag behind a version.**

**Comment by AllenWu233** [comment]
The release works well on Arch Linux.

**Reply by XurxoMF** [reply]
Perfectttt, thx for testing it ^^
